src/ebay-tourtip/ebay-tourtip-footer.tsx

```diff
@@ -0,0 +1,14 @@
+import { FC, ComponentProps } from 'react'
```

Comment:
This component should be responsible for handling the `index` logic and render the `children`. Inside the `EbayTourtip` we will render this component and the base tooltip footer:

```jsx

footer && (
  <TooltipFooter type="tourtip">
     {footer}
  </TooltipFooter>
)
```